lib/new_relic/agent/instrumentation/active_job.rb

```diff
@@ -71,7 +71,7 @@ def self.perform(job, block)
         end
 
         def self.run_in_trace(job, block, event)
-          trace_execution_scoped("MessageBroker/#{adapter}/Queue/#{event}/Named/#{job.queue_name}",
+          trace_execution_scoped("ActiveJob/#{adapter}/Queue/#{event}/Named/#{job.queue_name}",
```

Comment:
Given the resulting change:

```diff
- MessageBroker/ActiveJob::Inline/Queue/Produce/MyQueueName
+ ActiveJob/ActiveJob::Inline/Queue/Produce/MyQueueName
```

I'm questioning the double `ActiveJob` string appearances. Maybe we should trim the leading `ActiveJob::` string off of the adapter:

```ruby
adapter.sub(/^ActiveJob::/, '')
```

Reply:
I like this! The repetition isn't necessary. [I've made this edit](https://github.com/newrelic/newrelic-ruby-agent/pull/1811/commits/3f7f7d96cc73dde89866d5de362dba65429f8421) - thanks for providing the regex.